src/dbnode/storage/database.go

```diff
@@ -236,10 +236,18 @@ func (d *db) updateSchemaRegistry(newNamespaces namespace.Map) error {
 		curSchema, err := schemaReg.GetLatestSchema(metadata.ID())
 		if curSchema != nil {
 			curSchemaId = curSchema.DeployId()
+			if len(curSchemaId) == 0 {
+				d.log.Warn("can not update namespace schema to with empty deploy ID", zap.Stringer("namespace", metadata.ID()),
```

Comment:
"can not update namespace schema with an empty deploy ID"

Reply:
ok